### ISSUE TITLE
New docker images aren't being created on releases

### ISSUE THREAD

**Issue by cbaker6** [parse-community/parse-server#8904]
### New Issue Checklist

- [x] I am not disclosing a [vulnerability](https://github.com/parse-community/parse-server/blob/master/SECURITY.md).
- [x] I am not just asking a [question](https://github.com/parse-community/.github/blob/master/SUPPORT.md).
- [x] I have searched through [existing issues](https://github.com/parse-community/parse-server/issues?q=is%3Aissue).
- [x] I can reproduce the issue with the [latest version of Parse Server](https://github.com/parse-community/parse-server/releases). 

### Issue Description

Docker images stopped deploying on releases for 3+ months https://hub.docker.com/r/parseplatform/parse-server/tags. Instead, the dock CI hangs.

I have a workaround for the issue, will submit shortly, as the problem consist of all a couple of items I've seen in the docker community. The parse-dashboard repo doesn't have this issue because GitHub actions in that repo is only building for the `linux/amd64` platform which doesn't require QEMU or buildx.

### Steps to reproduce

Attempt to do a new release.

### Actual Outcome

No new image is created

### Expected Outcome

A new image on docker hub

### Environment

Server
- Parse Server version: `6.4.0` and `alpha`
- Operating system: `Linux`
- Local or remote host (AWS, Azure, Google Cloud, Heroku, Digital Ocean, etc): `local`

Database
- System (MongoDB or Postgres): `N/A`
- Database version: `N/A`
- Local or remote host (MongoDB Atlas, mLab, AWS, Azure, Google Cloud, etc): `N/A`

Client
- SDK (iOS, Android, JavaScript, PHP, Unity, etc): `N/A`
- SDK version: `N/A`

### Logs

See https://github.com/parse-community/parse-server/actions/runs/7616018905/job/20741904342

**Comment by parseplatformorg** [comment]
🎉 This change has been released in version [7.0.0-alpha.13](https://github.com/parse-community/parse-server/releases/tag/7.0.0-alpha.13)

**Comment by mtrezza** [comment]
Reopneing; building docker image fails, see for example https://github.com/parse-community/parse-server/actions/runs/7907865797/job/21585886190:

> Error: buildx failed with: ERROR: failed to solve: process "/bin/sh -c npm ci --omit=dev --ignore-scripts  && cp -R node_modules prod_node_modules  && npm ci  && npm run build" did not complete successfully: exit code: 1

**Comment by cbaker6** [comment]
Taking a look and will report back with a fix...

**Comment by cbaker6** [comment]
My initial guess is this error might be random, I see 7.0.0-alpha.13, https://hub.docker.com/layers/parseplatform/parse-server/7.0.0-alpha.13/images/sha256-dc68755c92c66ea70c2437cdc59ced526a0f82d9fdfbfdfb55ecf731785cdb50?context=explore, created an image which was the release right after #8905 was merged.

I’ll wait to see if it fails consistently, but looks like a random network issue https://github.com/parse-community/parse-server/actions/runs/7907865797/job/21585886190#step:8:705

**Comment by mtrezza** [comment]
Restarted all [failed docker jobs ](https://github.com/parse-community/parse-server/actions/workflows/release-automated.yml) since the fix and they all failed again. Also tried the [manual workflow](https://github.com/parse-community/parse-server/actions/runs/7915413843). Maybe the chained command could be broken up to get a more detailed error?

**Comment by parseplatformorg** [comment]
🎉 This change has been released in version [7.0.0-alpha.24](https://github.com/parse-community/parse-server/releases/tag/7.0.0-alpha.24)

**Comment by parseplatformorg** [comment]
🎉 This change has been released in version [6.5.3](https://github.com/parse-community/parse-server/releases/tag/6.5.3)

**Comment by parseplatformorg** [comment]
🎉 This change has been released in version [7.0.0-beta.1](https://github.com/parse-community/parse-server/releases/tag/7.0.0-beta.1)

**Comment by parseplatformorg** [comment]
🎉 This change has been released in version [7.0.0](https://github.com/parse-community/parse-server/releases/tag/7.0.0)
